### ISSUE TITLE
`order`'s `newlines-between` doesn't work with multiline import statements

### ISSUE THREAD

**Issue by singles** [benmosher/eslint-plugin-import#313]
Settings `newlines-between: always`

Not working examples:

```js
import FurtherStepsHeadline // invalid error report here: There should be no empty line within import group  import/order
   from '../../../../../../app/components/pages/contentFunnel/donePartials/FurtherStepsHeadline';
```

```js
import { expect } from 'chai';

import RangeType, { // invalid error report here: There should be no empty line within import group  import/order
    isFullRange,
    isPartialRange,
} from '../../../../app/lib/dateRange';
```

```js
local: {
	launch_url: 'http://localhost:${APPLICATION_PORT}',
	selenium_host: 'localhost',
	selenium_port: 4444,
	selenium: {
		start_process: true,
		server_path: require('selenium-server-standalone-jar').path, // invalid error report here: There should be no empty line within import group  import/order
		cli_args: {
			'webdriver.chrome.driver': require('chromedriver').path
		}
	},
	desiredCapabilities: {
		browserName: 'chrome'
	}
},
```

I will fix this ASAP.

**Comment by benmosher** [comment]
For that last one: might want to iterate `body` in `Program` + `BlockStatement` visitors vs. checking all `CallExpressions`.

I think that would match what people expect for `require`. Also, should be able to reuse the same visitor implementation for both, each has a `body` Array of statements.

**Comment by jfmengels** [comment]
I thought of another case when `newlines-between: "never"`:

```js
import { expect } from 'chai';
import 'loud-rejection';
import _ from 'lodash';
```

In this case, `loud-rejection` will be ignored as it isn't assigned to a variable (see https://github.com/jfmengels/eslint-plugin-import-order/issues/5 for the discussion if you're interested). I haven't tested it, but I have a suspicion that this would trigger an error too.

Could you add test for it (and fix it if it breaks)?

**Comment by singles** [comment]
@jfmengels Yeah, I'll have a look.

**Comment by benmosher** [comment]
@singles Should this be closed now that 339 is merged?

**Comment by jfmengels** [comment]
Yes :)

**Comment by benmosher** [comment]
👍
